### PR TITLE
Fixes #667

### LINKED ISSUE #667: Add note about windows option for install script
per feedback from @thesteve0, add a note in the Creating launch.openshift.io Application Using Automated Script that it does not support windows and redirect to manual section

### PR DESCRIPTION
**Pull request by rhoads-zach**
@thesteve0 @inoxx03 let me know what you think.

### CODE REVIEW

File: docs/topics/minishift-install-create-launchpad-app-script.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@ You can install the {launcher} application to your {OpenShiftLocal} using link:{
 * Deploy the {launcher} application to that project using the link:{link-launcher-yaml}[latest template].
 * Configure the {launcher} application with your GitHub username and personal access token.
 
+IMPORTANT: If you are running your {OpenShiftLocal} in Windows, you must the xref:launcher-manual-install[manual method] for installing the {launcher} application.
```
Comment (Ladicek) [inline docs/topics/minishift-install-create-launchpad-app-script.adoc]: "in Windows" -> "on Windows"? Dunno.

"you must the manual method" -> "you must use the manual method"

Comment (inoxx03) [reply docs/topics/minishift-install-create-launchpad-app-script.adoc]: personally, I'd be inclined to use "on Windows" in this context.

Comment (rhoads-zach) [reply docs/topics/minishift-install-create-launchpad-app-script.adoc]: thanks @Ladicek and @inoxx03. fixed

**Comment by inoxx03** [comment]
merging...

**Comment by inoxx03** [comment]
merged: bae9f67fd33241941052448e9af178e08b88860e

**Pull request merged**